erpnext/hr/doctype/full_and_final_outstanding_statements/full_and_final_outstanding_statements.json

```
@@ -0,0 +1,90 @@
+{
+ "actions": [],
+ "creation": "2021-06-28 13:32:02.167317",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "component",
+  "reference_document_type",
+  "reference_document",
+  "account",
+  "considered_in_salary_slip",
+  "column_break_4",
+  "status",
+  "amount",
+  "remark"
+ ],
+ "fields": [
+  {
+   "fieldname": "column_break_4",
+   "fieldtype": "Column Break"
+  },
+  {
+   "default": "Unsettled",
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "options": "Settled\nUnsettled"
+  },
+  {
+   "fieldname": "remark",
+   "fieldtype": "Small Text",
+   "label": "Remark"
+  },
+  {
+   "depends_on": "reference_document_type",
+   "fieldname": "reference_document",
+   "fieldtype": "Dynamic Link",
+   "in_list_view": 1,
+   "label": "Reference Document",
+   "mandatory_depends_on": "reference_document_type",
+   "options": "reference_document_type",
+   "search_index": 1
+  },
+  {
+   "fieldname": "component",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Component",
+   "reqd": 1
+  },
+  {
+   "fieldname": "account",
+   "fieldtype": "Link",
+   "label": "Account",
+   "options": "Account"
+  },
+  {
+   "fieldname": "amount",
+   "fieldtype": "Currency",
+   "in_list_view": 1,
+   "label": "Amount"
+  },
+  {
+   "default": "0",
+   "fieldname": "considered_in_salary_slip",
+   "fieldtype": "Check",
+   "label": "Considered In Salary Slip"
```

```suggestion
   "fieldname": "payable_via_salary_slip",
   "fieldtype": "Check",
   "label": "Payable via Salary Slip"
```

Let's rename this to Payable via Salary Slip instead?
"Considered" seems ambiguous when the actual meaning is this component will be paid via journal entry if this is disabled.

---

Actually, It should not be like  "payable_via_salary_slip". Actually, some of the components like leave Encashment and Incentives are already Covered in the Salary slip This label Is for that. Open for Better Label suggestion, we can Have a discussion about that.

---

How about **Paid via Salary Slip** then?

---

done